en/lessons/cleaning-data-with-openrefine.md

```diff
@@ -153,8 +153,8 @@ downloaded data file and click **Next**. By default, the first line will
 be correctly parsed as the name of a column, but you need to unselect
 the 'Quotation marks are used to enclose cells containing column
 separators' checkbox, since the quotes inside the file do not have any
-meaning to *OpenRefine*. Additionally, select the 'Parse cell text into
-numbers, dates, ...' checkbox to let OpenRefine automatically detect
+meaning to *OpenRefine*. Additionally, select the 'Attempt to parse cell
+text into numbers' checkbox to let OpenRefine automatically detect
```

Comment:
This reflects an update that we made in the tool, changing the name of this option to match its actual behavior (it doesn't actually parse dates).